pkg/virt-handler/node-labeller/node_labeller_test.go

```diff
@@ -87,22 +90,30 @@ var _ = Describe("Node-labeller ", func() {
 					MinCPUModel:       "Penryn",
 				},
 			},
-		}
-
-		initNodeLabeller(kv)
+		})
+		mockQueue.ExpectAdds(1)
```

Comment:
can you explain why you moved this code to the end of the BeforeEach in commit `Fix node labeller tests`? 
If there's no "big" reason it should be on this commit - then I suggest to move this to the first "cleanup" commit.